### DISCUSSION TITLE
How to add meta description tags?

### DISCUSSION THREAD

**Discussion by talonx** [GitbookIO/community#695] [1. How to]
I looked at https://docs.gitbook.com/published-documentation/seo#seo-friendly-content but was not able to find any documentation on how to add a 
```
<meta name="description" > 
```
tag to my book. 

Is there any option to do this?

**Comment by Azamat4985** [comment]
Hello! Did you find any solution?

**Reply by talonx** [reply]
> Hello! Did you find any solution?

I was able to add tags to individual pages by setting the page "description"s. The option to add this shows up in the editor.
